src/block_cache/block_io.c

```diff
@@ -820,10 +867,12 @@ __wt_blkcache_write(WT_SESSION_IMPL *session, WT_ITEM *buf, WT_PAGE_BLOCK_META *
         WT_STAT_CONN_INCR(session, block_cache_bypass_chkpt);
     else if (!blkcache->cache_on_writes)
         WT_STAT_CONN_INCR(session, block_cache_bypass_writealloc);
-    else if (!checkpoint)
+    else if (!checkpoint) {
```

Comment:
This is quite a confusing line, could you please spell it out a bit more in the comment.

Reply:
ip, ctmp, and buf are all items representing more or less different representations of the same block. It's not clear by the names which WT_ITEM represents what.

Reply:
Hmm, actually `ctmp` and `etmp` are used elsewhere in the code too. Renaming them feels more like a separate refactoring task.